Action connectors can be imported/exported with rules (#3021)
elastic/security-docs: docs/getting-started/detections-req.asciidoc

* To provide full access to rule actions and connectors, give your role `All` privileges. With `Read` privileges, you can edit rule actions, but will have limited capabilities to manage connectors. For example, `Read` privileges allow you to add or remove an existing connector from a rule, but does not allow you to create a new connector.

> * To import rules with actions, you need at least `Read` privileges for the `Action and Connectors` feature. To overwrite existing connectors or add new connectors, you need `All` privileges for the `Action and Connectors` feature. If you're importing rules without actions, `Action and Connectors` feature privileges are not required. 

Reviewer: ```suggestion
* To import rules with actions, you need at least `Read` privileges for the `Actions and Connectors` feature. To overwrite existing connectors or add new connectors, you need `All` privileges for the `Action and Connectors` feature. If you're importing rules without actions, `Actions and Connectors` feature privileges are not required. 
```
Same as above

Author: ```suggestion
* To import rules with actions, you need at least `Read` privileges for the `Action and Connectors` feature. To overwrite or add new connectors, you need `All` privileges for the `Actions and Connectors` feature. To import rules without actions,  you don't need `Actions and Connectors` privileges. Refer to <<enable-detections-ui>> for more information. 
```

> * To import rules with actions, you need at least `Read` privileges for the `Action and Connectors` feature. To overwrite or add new connectors, you need `All` privileges for the `Actions and Connectors` feature. To import rules without actions,  you don't need `Actions and Connectors` privileges. Refer to <<enable-detections-ui>> for more information. 

Reviewer 2: Not sure the cross-ref link is necessary here, since it basically points to the same page & section that this content is in.

Author: Good point, it's not necessary. 

```suggestion
* To import rules with actions, you need at least `Read` privileges for the `Action and Connectors` feature. To overwrite or add new connectors, you need `All` privileges for the `Actions and Connectors` feature. To import rules without actions,  you don't need `Actions and Connectors` privileges. 
```
